required/latex-lab/latex-lab-block.dtx

```diff
@@ -2210,12 +2298,26 @@
       \tl_set_eq:NN \@itemlabel \l_@@_item_label_tl
     }
 %    \end{macrocode}
-%    finally, we signal that we are at the start of a new list (which
+%    Finally, we signal that we are at the start of a new list (which
 %    effects how the first \cs{item} is handled and how \cs{par}
```

Comment:
"affects" is (I think!) needed here: effect is a noun only(??), the verb is "affect".